hasher-matcher-actioner/scripts/soak_test_system

```diff
@@ -46,32 +53,137 @@ CLIENT_ID = ""
 LISTENER_EXTERNAL_HOSTNAME = ""
 LISTENER_PORT = 8080
 
+ID_SEPARATOR = "-"
+
+
+class SubmittingThread(threading.Thread):
+    def __init__(
+        self,
+        helper: DeployedInstanceTestHelper,
+        submission_batch_size=10,
+        seconds_between_batches=10,
+        **kwargs,
+    ):
+        super(SubmittingThread, self).__init__(**kwargs)
+        self.daemon = True
+        self._stop_signal = threading.Event()
+        self.submission_batch_size = submission_batch_size
+        self.seconds_between_batches = seconds_between_batches
+        self.helper = helper
+
+    def stop(self):
+        self._stop_signal.set()
+
+    def stopped(self):
+        return self._stop_signal.is_set()
+
+    def run(self):
+        total_submitted = 0
+        while True:
+            if self.stopped():
+                print(f"Submissions stopped. TOTAL SUBMITTED: {total_submitted}")
+                return
+            batch_prefix = f"soak_test{ID_SEPARATOR}{datetime.date.today().isoformat()}{ID_SEPARATOR}{str(uuid.uuid4())}"
+            for i in range(self.submission_batch_size):
+                content_id = f"{batch_prefix}-{i}"
+                self.helper.submit_test_content(content_id)
+                total_submitted += 1
+            print(f"Submitter: total submission count {total_submitted}")
```

Comment:
Dunno how you feel about it, but if you do ` print("some string", end='\r')`, you can 'update' the line with the new count instead of adding more screen output.

Reply:
May not be possible because you are interleaving text output from different threads.

Reply:
Yeah, I think threading breaks this as cool as it would be.

Reply:
Leaving as is for now but will try it out as part of improvements.